CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing guidelines
+
+Welcome! We're glad to see you here. This short document should give you some hints to get started with contributing to TermPair.
+
+## Getting started
+
+First, fork the repo and clone it to your computer, then read the section you're interested in. 👍
+
+### Contributing Python code
+
+To modify Python code, setup a virtual environment:
+
+```bash
+python -m venv venv
+```
+
+Then activate it with:
+
+```bash
+. venv/bin/activate
+```
+
+You can then install development dependencies using:
+
+```bash
+pip install -e ".[dev]"
```

Comment:
I don't think the quotes are necessary around `.[dev]`

Reply:
They were on my machine (`zsh: no matches found: .[dev]`) so I assumed this would be the case for most terminal setups. I'll remove it if it works without quotes on your end. :-)

Reply:
Oh, if you needed them then they should be left. They don't hurt anything.